packages/ndla-ui/src/LicenseByline/EmbedByline.tsx

```diff
@@ -197,6 +197,7 @@ const TextContent = styled("span", {
       transitionProperty: "max-height",
       transitionDuration: "slow",
       transitionTimingFunction: "ease-in",
+      marginInlineEnd: "4xsmall",
```

Comment:
Dette skal vel ikke være `end`?

Reply:
Det er space på slutten av teksten før knappen. Om det var `Start` så hadde det ikke blitt noe endring mellom tekst og knappen

Reply:
Ah, skjønner. Gir det isåfall mening å sette `marginInlineStart` på knappen istedenfor? Da setter vi bare margin dersom den faktisk er der.